### PR TITLE
Fixed some notebook typos

### PR DESCRIPTION
**Pull request by tgrnie**
Fixed lots of typos.

There was also an uncessary use of `list` wrapped around `range_of`.

### CODE REVIEW

File: nbs/13a_learner.ipynb [hunk 1]
```diff
@@ -3583,7 +3583,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In practice, we get the predictions `n` times with the transforms of the training set and average those. The final predictions are `(1-beta)` multiplied by this average + `beta` multiplied by the predictions obtained with the transforms of the dataset. Set `beta` to `None` to get a tuple of the predictions and tta results. You can also use the maximum of all predictions instead of an average by setting `use_max=True`.\n",
+    "In practice, we get the predictions `n` times with the transforms of the training set and average those. The final predictions are `(1-beta)` multiplied by this average + `beta` multiplied by the predictions obtained with the transforms of the dataset. Set `beta` to `None` to get a tuple of the predictions and the results. You can also use the maximum of all predictions instead of an average by setting `use_max=True`.\n",
```
Comment (jph00) [inline nbs/13a_learner.ipynb:3586]: I believe that this should be left as "tta"

**Review by jph00 — CHANGES_REQUESTED** [review]
Awesome thanks! Just one requested change, if that's OK

File: nbs/13a_learner.ipynb (hunk 1 above)
Comment (tgrnie) [reply nbs/13a_learner.ipynb:3586]: Sorry about that, will change now!

**Comment by jph00** [comment]
That was fast! :)

**Pull request merged**